containers/wordpress/wp_opt/https/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG baseimage=wp_http_opt_with_https
+FROM $baseimage
+
+LABEL authors="nitin.tekchandani@intel.com,lin.a.yang@intel.com,uttam.c.pawar@intel.com"
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y libtool
+
+# Build/Install IPP cyrypto library for the multi-buffer support
+WORKDIR /home/${USERNAME}/temp
+RUN git clone https://github.com/intel/ipp-crypto.git && \
+    cd ipp-crypto && \
+    git checkout master && \
+    cd sources/ippcp/crypto_mb && \
+    CXX=g++-9 CC=gcc-9 cmake . -B./build -DOPENSSL_INCLUDE_DIR=/home/${USERNAME}/openssl_install/include/ -DOPENSSL_LIBRARIES=/home/${USERNAME}/openssl_install/ -DOPENSSL_ROOT_DIR=/home/${USERNAME}/temp/openssl/ && \
+    cd build && \
+    make -j && \
+    mkdir -p /home/${USERNAME}/crypto_mb/lib && \
+    ln -s /home/${USERNAME}/temp/ipp-crypto/sources/ippcp/crypto_mb/build/bin/libcrypto_mb.so /home/${USERNAME}/crypto_mb/lib/ && \
+    ln -s /home/${USERNAME}/temp/ipp-crypto/sources/ippcp/crypto_mb/include /home/${USERNAME}/crypto_mb/include
+
+# Build/Install QAT engine 
+WORKDIR /home/${USERNAME}/temp
+RUN git clone  https://github.com/intel/QAT_Engine.git qat_engine && \
+    cd qat_engine && \
+    git checkout v0.6.1 && \
```

Comment:
Suggest having versions for these instead of hardcoding.

Reply:
done